generators/fastlane-setup/templates/fastlane/Fastfile

```diff
@@ -161,13 +172,16 @@ platform :ios do
       if ENV['DEPLOYMENT_PLATFORM'] === 'hockeyapp' then
         deploy_hockey
       elsif ENV['DEPLOYMENT_PLATFORM'] === 'appcenter' then
-        deploy_appCenter
+        upload_to_appcenter
       else
         pilot(
           distribute_external: false,
           skip_waiting_for_build_processing: true
         )
       end
+      if ENV['IOS_APPSTORECONNECT_USER_ID'] then
```

Comment:
it's in the "else" above

Reply:
that else should really be `elsif ENV['DEPLOYMENT_PLATFORM'] === 'appstore' then`